File: test/py/case31_inheritance.py
class Base31:
    def value(self) -> int:
        return 31


class Child31(Base31):
    def value2(self) -> int:
        return self.value() + 1


if __name__ == "__main__":
    c: Child31 = Child31()
    print(c.value2())

File: test/cs/case31_inheritance.cs
using System;

public static class Program
{
    public class Base31
    {
        public int value()
        {
            return 31;
        }
    }

    public class Child31 : Base31
    {
        public int value2()
        {
            return (this.value() + 1);
        }
    }

    public static void Main(string[] args)
    {
        Child31 c = new Child31();
        Console.WriteLine(c.value2());
    }
}
